opentreemap/treemap/templates/instance_base.html

```diff
@@ -158,9 +161,14 @@
            disabled="disabled"><i class="icon-plus"></i> {% trans "Add a" %} {{ term.Resource.singular }}</a>
         {% endif %}
       <a class="btn btn-primary addBtn"
-         data-action='addtree'
+         {% if embed %}
+         target="_blank"
+         {% else %}
          data-feature="add_plot"
```

Comment:
No change needed. I was just verifying that these attributes were being hidden by design.